pkg/subscriptions/subscriptions.js

```diff
@@ -433,6 +440,9 @@ require([
                     } else if (buffer.indexOf('Unable to reach the server at') === 0) {
                         reason = "server";
                         message = buffer.trim();
+                    } else if ((buffer.indexOf('EOF') === 0) && (buffer.indexOf('Organization: ') !== -1)) {
```

Comment:
The error message literally begins with the string `EOF`, since we use CLI scraping for this and `subscription-manager` wants to have user input. As a second condition, we want to see if it asks for our organization. I think the complete message contains some other stuff I wasn't comfortable with checking against in its entirety.